src/main/java/command/AddCommand.java

```diff
@@ -0,0 +1,35 @@
+package command;
+import task.*;
```

Comment:
Imported classes should always be listed explicitly. [here](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements). May be can set your IDE settings to not automatically convert to wildcard imports.
```suggestion
import task.Task;
```